game-core/src/main/java/games/strategy/triplea/ui/AbstractForumPosterPanel.java

```diff
@@ -78,21 +66,21 @@ protected void waitForDone(final TripleAFrame frame, final IPlayerBridge bridge)
     if (skipPosting() || GameStepPropertiesHelper.isSkipPosting(getData())) {
       return;
     }
-    final boolean hasPosted = getHasPostedTurnSummary();
+
+    final boolean hasPosted =
+        ((IAbstractForumPosterDelegate) playerBridge.getRemoteDelegate()).getHasPostedTurnSummary();
```

Comment:
Inlined this method, used to be an abstract, both child classes had the same implementation.